### PR TITLE
feat: example of Equinix Metal deployment

### PR DESCRIPTION
**Pull request by rawkode**
This example uses a Global IP via BGP to allow low-latency routing to multiple platform servers

### CODE REVIEW

File: equinix-metal/terraform/variables.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+variable "project_id" {
```
Comment (vdice) [inline equinix-metal/terraform/variables.tf:1]: This var doesn't have a default, but I don't seem to see a reference to it in the README.md -- can/should we show how it should be set/used?

File: equinix-metal/README.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+service.
+
+That being said, it should give users a quick look and feel for deploying apps
+using Fermyon. By default, all apps will be accessible to the broader internet
+(see the configuration details mentioned below). Additionally, when Let's Encrypt
+is enabled, apps will be provided with https URLs and TLS certs.
+
+# Prerequisites
+
+- An Equinix Metal account with a user level API key. See [here](https://console.equinix.com/profile/api-keys) for more details.
```
Comment (vdice) [inline equinix-metal/README.md:19]: I signed up for an Equinix Metal account but am having trouble finding if there are any trial or free usage options -- any chance there are?

**Review by vdice — APPROVED** [review]
Code looks good to me!  I haven't yet tested it myself.

@rawkode it looks like the last commit is missing the DCO -- can you add/amend when you have a chance?

**Review by vdice — COMMENTED** [review]
Another optional request: Shall we add a link to this new guide in the root [README](https://github.com/fermyon/installer/blob/main/README.md)?

File: equinix-metal/README.md (hunk 2 above)
Comment (rawkode) [reply equinix-metal/README.md:19]: You can use the code "rawkode" for 200USD credits

**Review by vdice — APPROVED** [review]
Update to mention that I was able to deploy Fermyon Platform to Equinix Metal from this PR.  All services were running/healthy and an example app deployed as expected.  Thanks @rawkode!

**Pull request merged**